dask_searchcv/model_selection.py

```diff
@@ -109,6 +109,16 @@ def build_graph(estimator, cv, scorer, candidate_params, X, y=None,
                        n_splits, error_score, weights)
     keys = [cv_results]
 
+    if not _is_arraylike(refit) and refit:
+        refit = True
+    elif _is_arraylike(refit):
+        if isinstance(refit, (tuple, list)) and len(refit) == 2:
```

Comment:
This is to disambiguate the `X` vs `(X, y)` cases?

Reply:
The `if not _is_arraylike(refit) and refit` block handles the current usage cases where `refit` is truthy and not a `Sequence`.
The `elif _is_arraylike(refit):` handles the case `refit` has been passed not as a boolean but as `X` or `(X, y)` that can go to the best estimator's `fit` method.  Note this PR doesn't attempt to handle passing of `y` through every step of a `Pipeline`, as we discussed on #61 .  This quoted block regarding `refit` being array-like is needed if the user passes `cache_cv` as a CVCache-like instance (otherwise they get an exception on fitting the best estimator if `refit=True`)